lib/google_drive_v0.rb

```diff
@@ -6,143 +6,126 @@
 require "json"
 
 require "google_drive_v0/session"
-
+require "shared/config"
```

Comment:
Sorry for taking time to modify this file, but these "v0" version no longer works, so you can just leave it. Can you revert the change for this file? I should delete these files...